djconnectwise/models.py

```diff
@@ -367,6 +367,29 @@ def __str__(self):
         return self.name
 
 
+class Contact(models.Model):
+
+    def __str__(self):
+        return '{} {}'.format(self.first_name,
+                              self.last_name if self.last_name else '')
+
+    first_name = models.TextField(blank=True, null=True, max_length=30)
+    last_name = models.TextField(blank=True, null=True, max_length=30)
+    company = models.ForeignKey(
+        'Company', null=True, on_delete=models.CASCADE)
+    address_line1 = models.CharField(blank=True, null=True, max_length=250)
```

Comment:
@kti-matt do we want to sync address, city, state, zip? Would they be used in the ticket notification templates later on?

Reply:
The issue requirements say to sync the address. But actually it might be better not to right now, because we have no plans to use those details right now and there could potentially be personally identifying information there. So actually let's not sync it or have fields for it.

Reply:
ok, thanks.